frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install.tsx

```diff
@@ -73,9 +81,9 @@ export const CreateAttachedDevicesCluster: React.FC<CreateAttachedDevicesCluster
       .catch(() => setHasNoProvSC(false));
   }, [appName, ns]);
 
-  return !allDataLoaded && !subscriptionLoadError ? (
+  return !isDataLoaded.current && !subscriptionLoadError && !csvLoadError ? (
     <LoadingBox />
-  ) : subscriptionLoadError || !isLsoPresent ? (
+  ) : subscriptionLoadError || csvLoadError || !lsoSubscription || !isLsoCsvSucceeded ? (
```

Comment:
This should be suffice
```suggestion
  ) : subscriptionLoadError || csvLoadError || !isLsoCsvSucceeded ? (
```
```suggestion
  ) : subscriptionLoadError || csvLoadError || !lsoSubscription || !isLsoCsvSucceeded ? (
```